docs/admin/kube-scheduler.md

```diff
@@ -62,6 +62,10 @@ kube-scheduler
       --kube-api-burst=100: Burst to use while talking with kubernetes apiserver
       --kube-api-qps=50: QPS to use while talking with kubernetes apiserver
       --kubeconfig="": Path to kubeconfig file with authorization and master location information.
+      --leader-elect[=false]: Start a leader election client and gain leadership before executing scheduler loop. Enable this when running replicated schedulers.
+      --leader-elect-lease-duration=15s: Lease duration is the duration that non-leader candidates will wait until attempting to acquire leadership of a led but unrenewed leader slot. This is effectively the maximum duration that a leader can be stopped before it is replaced by another candidate. This is only applicable if leader election is enabled.
+      --leader-elect-renew-deadline=10s: Renew deadline is the duration that the acting master will retry renewing a leadership slot before it stops leadeing. This must be less than or equal to the lease duration. This is only applicable if leader election is enabled.
+      --leader-elect-retry-period=2s: Retry period is the duration the LeaderElector clients should wait between actions against the API. Common actions are attempting to aquire the leader slot, renewing  leadership of a leader slot, or observing any updates of a leader slot. This is only applicable if leader election is true.
```

Comment:
this sentence could probably be trimmed : 
Retry period is the duration the clients should wait between lease acquisition and renewals.  This is only applicable if leader election is true.